docs/ocis/release_notes.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Release Notes"
+date: 2020-12-16T20:35:00+01:00
+weight: 0
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis
+geekdocFilePath: release_notes.md
+---
+
+## ownCloud Infinite Scale 1.0.0 Technology Preview - Initial Release Notes
+
+We are pleased to annouce the availability of ownCloud Infinite Scale 1.0.0 Technology Preview which is released as the first public version of the new Infinite Scale platform.
+
+### Microservice architecture
+
+ownCloud Infinite Scale is following the microservices architectual pattern. It is implemented as a set of microservices which are independent of each other. They are coupled with very well-defined APIs and communicate via HTTP. This architecture fosters a lot of benefits that we were going for with the new design for oCIS:
+
+- Independent services: Every service is independant, comparably small and brings it's own webserver, backend/APIs and frontend components.
+- Each service runs as a separate service on the system, increasing security and stability
+- Scalability:  High performance demands can be fulfilled by scaling the amount of services
+- Testability: Each service can be tested on its own due to well-defined APIs and functionality
+- Protocol-driven development
+- High-performance communication between services through technologies like GRPC
+- Multi-platform support through utilizing Golang - only minimal dependency on platform packages.
+- Cloud-native deployment and update strategies
+
+### Key figures
+
+- The all-new ownCloud Web frontend ships with the platform
+- OpenID Connect is the technology choice for authentication
+- An Identity Provider is bundled to ease deployment and operations. It can be replaced with other applications if desired.
+- Up-to-date, cloud-native deployment options are available
+- Flexible configuration through environment variables, yaml files or command-line switches
+- Database-less architecture - metadata and data are kept together in the storage as a single source of truth
+- Native storage capabilities are used where possible
+- Public ownCloud APIs like WebDAV and OCS have been kept compatible to ownCloud 10
+- A secure and flexible framework to create extensions for ownCloud. It allows to integrate with ownCloud data in a very easy yet powerful way.
+
+#### Supported platforms
+
+- Linux-amd64
+- Darwin-amd64
+- Experimental: Windows, ARM (e.g., Raspberry Pi)
+
+#### Client support
+
+All official ownCloud Clients support the Infinite Scale server with the following versions:
+- Desktop >= 2.7.0
```

Comment:
```suggestion
- Desktop >= 2.7
```

Maybe just document the minor version here, like done for Android and iOS below. And people get the latest patch version anyway (2.7.3, 2.7.4, whatever...)